tests/baselines/reference/assignmentLHSIsValue.errors.txt

```diff
@@ -109,6 +110,8 @@ tests/cases/conformance/expressions/assignmentOperator/assignmentLHSIsValue.ts(7
     
     // array literals
     ['', ''] = value;
+    ~~~~~~~~
+!!! error TS2461: Type 'any' is not an array type.

```

Comment:
Weird

1. It should be okay with `any`, you shouldn't get an error from this, should you?
2. That shouldn't even be a valid LHS.